scripts/facts-tracker/index.js

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var fs = require('fs');
+var execSync = require('child_process').execSync;
+
+function escape(value) {
+  return '\'' + value.replace(/'/g, "'\\''") + '\'';
+}
+
+function exec(command) {
+  console.log('>', command.replace(process.env.GITHUB_TOKEN, '************'));
+  return execSync(command).toString();
+}
+
+if (process.env.TRAVIS_REPO_SLUG) {
+  if (process.env.TRAVIS_BRANCH !== 'master') {
+    console.error('facts-tracker: Branch is not master, exiting...');
+    process.exit(1);
+  }
+
+  if (process.env.TRAVIS_PULL_REQUEST !== 'false') {
+    console.error('facts-tracker: This is a pull request, exiting...');
+    process.exit(1);
+  }
+
+  if (!process.env.GITHUB_USER ||
+      !process.env.GITHUB_TOKEN) {
+    console.error(
+      'In order to use facts-tracker, you need to configure a ' +
+      'few environment variables in order to be able to commit to the ' +
+      'repository. Follow those steps to get you setup:\n' +
+      '\n' +
+      'Go to https://github.com/settings/tokens/new\n' +
+      ' - Fill "Token description" with "facts-tracker for ' + process.env.TRAVIS_REPO_SLUG + '"\n' +
+      ' - Check "public_repo"\n' +
+      ' - Press "Generate Token"\n' +
+      '\n' +
+      'In a different tab, go to https://travis-ci.org/' + process.env.TRAVIS_REPO_SLUG + '/settings\n' +
+      ' - Make sure "Build only if .travis.yml is present" is ON\n' +
+      ' - Fill "Name" with "GITHUB_TOKEN" and "Value" with the token you just generated. Press "Add"\n' +
+      ' - Fill "Name" with "GITHUB_USER" and "Value" with the name of the account you generated the token with. Press "Add"\n' +
+      '\n' +
+      'Once this is done, commit anything to the repository to restart Travis and it should work :)'
+    );
+    process.exit(1);
+  }
+
+  exec(
+    'echo "machine github.com ' +
+    'login ' + escape(process.env.GITHUB_USER) + ' ' +
+    'password ' + escape(process.env.GITHUB_TOKEN) +
+    '" > ~/.netrc'
+  );
+  var remoteOriginChanged = true;
+  exec('git remote rename origin __old_origin');
+  exec(
+    'git remote add origin https://' + escape(process.env.GITHUB_TOKEN) +
+    '@github.com/' + escape(process.env.TRAVIS_REPO_SLUG)
+  );
+  exec(
+    'git config --global user.name ' +
+    escape(process.env.GITHUB_USER_NAME || 'facts-tracker')
```

Comment:
Can you make sure we have these setup first. We already have a user we're using for the docs updates and it shows as "Travis CI" - I'd like this too as well.

I don't care about the fallback value you have here but I'm pretty sure we don't have these set right now. When you add the values we have for the docs, we might as well update the docs command to use the env var as well.